src/test/java/staffconnect/logic/parser/AddMeetingCommandParserTest.java

```diff
@@ -0,0 +1,107 @@
+package staffconnect.logic.parser;
+
+
+import static staffconnect.logic.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+import static staffconnect.logic.commands.CommandTestUtil.DESC_MIDTERM;
+import static staffconnect.logic.commands.CommandTestUtil.INVALID_DESCRIPTION;
+import static staffconnect.logic.commands.CommandTestUtil.INVALID_STARTDATE;
+import static staffconnect.logic.commands.CommandTestUtil.MEETDATE_STARTDATE;
+import static staffconnect.logic.commands.CommandTestUtil.VALID_MEETING;
+import static staffconnect.logic.parser.CliSyntax.PREFIX_DESCRIPTION;
+import static staffconnect.logic.parser.CliSyntax.PREFIX_STARTDATE;
+import static staffconnect.logic.parser.CommandParserTestUtil.assertParseFailure;
+import static staffconnect.logic.parser.CommandParserTestUtil.assertParseSuccess;
+import static staffconnect.testutil.TypicalIndexes.INDEX_FIRST_PERSON;
+import static staffconnect.testutil.TypicalIndexes.INDEX_SECOND_PERSON;
+
+import org.junit.jupiter.api.Test;
+
+import staffconnect.commons.core.index.Index;
+import staffconnect.logic.Messages;
+import staffconnect.logic.commands.AddMeetingCommand;
+import staffconnect.model.meeting.Description;
+import staffconnect.model.meeting.MeetDateTime;
+
+public class AddMeetingCommandParserTest {
+
+    private static final String MEETING_MISSING_INDEX = " meeting at finals";
+
+    private static final String MESSAGE_INVALID_FORMAT =
+        String.format(MESSAGE_INVALID_COMMAND_FORMAT, AddMeetingCommand.MESSAGE_USAGE);
+
+    private final AddMeetingCommandParser parser = new AddMeetingCommandParser();
+
+    @Test
+    public void parse_missingParts_failure() {
+        // no index specified
+        assertParseFailure(parser, MEETING_MISSING_INDEX, MESSAGE_INVALID_FORMAT);
```

Comment:
Any reason why you use `MEETING_MISSING_INDEX` instead of having `VALID_MEETING_DESC` in `CommandTestUtil` which could be reused in other test files?

Reply:
Fixed